### PR TITLE
some minor fixes

### PR DESCRIPTION
**Pull request by int88**
**What type of PR is this?**

> Documentation

**What does this PR do? Why is it needed?**

some minor fixes

**Which issues(s) does this PR fix?**

Fixes #

**Other notes for review**

### CODE REVIEW

File: beacon-chain/state/stategen/replay.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func (_ *State) replayBlocks(
 func (s *State) loadBlocks(ctx context.Context, startSlot, endSlot types.Slot, endBlockRoot [32]byte) ([]interfaces.SignedBeaconBlock, error) {
 	// Nothing to load for invalid range.
 	if endSlot < startSlot {
-		return nil, fmt.Errorf("start slot %d >= end slot %d", startSlot, endSlot)
+		return nil, fmt.Errorf("start slot %d > end slot %d", startSlot, endSlot)
```
Comment (prestonvanloon) [inline beacon-chain/state/stategen/replay.go:84]: I think this was correct. `>=` is the inverse of `<`

File: beacon-chain/blockchain/error.go [hunk 2]
```diff
@@ -14,7 +14,7 @@ var (
 	// errNilFinalizedCheckpoint is returned when a nil finalized checkpt is returned from a state.
 	errNilFinalizedCheckpoint = errors.New("nil finalized checkpoint returned from state")
 	// errNilJustifiedCheckpoint is returned when a nil justified checkpt is returned from a state.
-	errNilJustifiedCheckpoint = errors.New("nil finalized checkpoint returned from state")
+	errNilJustifiedCheckpoint = errors.New("nil justified checkpoint returned from state")
```
Comment (prestonvanloon) [inline beacon-chain/blockchain/error.go:17]: nice catch

File: beacon-chain/state/stategen/replay.go (hunk 1 above)
Comment (int88) [reply beacon-chain/state/stategen/replay.go:84]: startSlot should <= endSlot, but the error is `startSlot > endSlot`, right?

Comment (prestonvanloon) [reply beacon-chain/state/stategen/replay.go:84]: Perhaps we can flip the if statement on 83 to match the original error message.

```go
	if startSlot >= endSlot {
		return nil, fmt.Errorf("start slot %d >= end slot %d", startSlot, endSlot)
	}
```

Comment (int88) [reply beacon-chain/state/stategen/replay.go:84]: `SetStartSlot()` and `SetEndSlot()` are both inclusive, so if `startSlot == endSlot`, we still could get one block to load. 

The only place to call this function is 
```golang
blks, err := s.LoadBlocks(ctx, startState.Slot()+1, targetSlot, bytesutil.ToBytes32(summary.Root))
```
in `beacon-chain/state/stategen/getter.go`

if `startState.Slot() + 1 == targetSlot`, there still one block of slot to load.

I wonder if it is OK that `startSlot == endSlot`, so the following statement would be better?

```golang
	if startSlot > endSlot {
		return nil, fmt.Errorf("start slot %d > end slot %d", startSlot, endSlot)
	}
```

Comment (prestonvanloon) [reply beacon-chain/state/stategen/replay.go:84]: OK i understand now and I agree with your assessment. My suggestion `>=` is not correct.

Your suggestion looks good to me. It's a pure flip without any changes. Thanks!

**Comment by int88** [comment]
@prestonvanloon updated, PTAL:)

**Review by prestonvanloon — APPROVED** [review]
Thanks!!

**Pull request merged**